tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/xml/XMLLiteralTest.java

```diff
@@ -107,8 +107,6 @@ public void testXMLNegativeSemantics() {
                 "'(xml<xml:Text>|xml<xml:Comment>)', found 'xml:ProcessingInstruction'", 73, 96);
         BAssertUtil.validateError(negativeResult, index++, "incompatible types: expected " +
                 "'(xml:Text|xml:Comment)', found 'xml:Element'", 74, 34);
-        BAssertUtil.validateError(negativeResult, index++, "incompatible types: expected " +
```

Comment:
This was originally for

```ballerina
'xml:Text|'xml:Comment x16 = xml `<root> text1<foo>100</foo><foo>200</foo></root><?foo?>`;
```

for which we now get

```
ERROR [foo.bal:(1:35,1:82)] incompatible types: expected '(xml:Text|xml:Comment)', found 'xml:Element'
ERROR [foo.bal:(1:82,1:89)] incompatible types: expected '(xml:Text|xml:Comment)', found 'xml:ProcessingInstruction'
```

This isn't correct and could be misleading. 

IMO, this should result in a single error like

```
incompatible types: expected '(xml:Text|xml:Comment)', found 'xml<xml:Element|xml:ProcessingInstruction>'
```